openedx/core/djangoapps/content/block_structure/store.py

```diff
@@ -229,11 +230,13 @@ def _encode_root_cache_key(bs_model):
         BlockStructureModel or StubModel.
         """
         if config.STORAGE_BACKING_FOR_CACHE.is_enabled():
-            return str(bs_model)
-        return "v{version}.root.key.{root_usage_key}".format(
-            version=str(BlockStructureBlockData.VERSION),
-            root_usage_key=str(bs_model.data_usage_key),
-        )
+            return six.text_type(bs_model)
+
+        else:
+            return "v{version}.root.key.{root_usage_key}".format(
+                version=six.text_type(BlockStructureBlockData.VERSION),
+                root_usage_key=six.text_type(bs_model.data_usage_key),
+            )
```

Comment:
I believe we could safely drop the change from this file - we're already in Python 3.8.
From: 83811298d5b8ed65d6fc50a0ea5bb6acb9993d6c